app/views/api/v2/common_parameters/main.json.rabl

```diff
@@ -2,4 +2,4 @@ object @common_parameter
 
 extends "api/v2/common_parameters/base"
 
-attributes :created_at, :updated_at, :hidden_value
+attributes :created_at, :updated_at, :hidden_value?
```

Comment:
This is not for lookup_keys, the (global) parameters showed "*****" instead of actually showing if they are hidden or not, this fixes it.